File: bindings/js/src/parsers/AMFJSONLDParser.ts
/**
 * Created by antoniogarrote on 05/05/2017.
 */

import {BaseParser} from "./BaseParser";
import {Clojure} from "../Clojure";

/**
 * Wrapper class for the AMF OWL model parser, processes AMF model specification JSON-LD documents and generate the DocumentModel out of them
 */
export class AMFJSONLDParser extends BaseParser {
    protected parser(): any {
        return new Clojure.amf.__GT_APIModelParser();
    }
}

File: bindings/java/src/org/raml/amf/parsers/AMFJSONLDParser.java
package org.raml.amf.parsers;

/**
 * Created by antoniogarrote on 04/05/2017.
 */

/**
 * Wrapper class for the AMF OWL model parser, processes AMF model specification JSON-LD documents and generate the DocumentModel out of them
 */
public class AMFJSONLDParser extends BaseParser {

    @Override
    protected String parserConstructor() {
        return "->APIModelParser";
    }
}
